README.md

```diff
@@ -65,19 +65,19 @@ This will start the local Hugo server on port 1313. Open up your browser to <htt
 
 The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
 
-To update the reference pages for a new Kubernetes release (replace v1.20 in the following examples with the release to update to):
+To update the reference pages for a new Kubernetes release follow these steps:
 
-1. Pull the `kubernetes-resources-reference` submodule:
+1. Pull in the `api-ref-generator` submodule:
 
    ```bash
    git submodule update --init --recursive --depth 1
    ```
```

Comment:
This is the right command, but it actually updates _all_ submodules.

Reply:
Right -- I wasn't implying that the command was wrong, only that the named submodule doesn't exist anymore.

Reply:
Since this is in the API reference section, i think It's ok to refer to the specific, relevant submodule being brought in.

Reply:
Right, I didn't want to change the style of the step prose. We can revisit it, if that's what Tim's intention was (sorry, I didn't get that at first).

Reply:
Yes, there is more tidying to do in the file.